### ISSUE TITLE
AndroidX migration

### ISSUE THREAD

**Issue by gianlucaparadise** [lbugnion/mvvmlight#80]
Is it planned a migration from Android Support to AndroidX for the `GalaSoft.MvvmLight.Platform.AndroidSupport` package?

In my opinion, this is a really blocking issue because it blocks the migration to AndroidX. The migration is necessary in order to use the latest libraries for the Android Ecosystem.

**Comment by MichaelShapiro** [comment]
@gianlucaparadise sorry for unrelated question, can you explain how exactly do you use the GalaSoft.MvvmLight.Platform.AndroidSupport package, please?

**Comment by gianlucaparadise** [comment]
@MichaelShapiro for example, we use the `GetRecyclerAdapter` method in order to generate an `ObservableRecyclerAdapter`. Then we set it to a `RecyclerView`.

Since the `ObservableRecyclerAdapter` [extends](https://github.com/lbugnion/mvvmlight/blob/bb9436e28a/GalaSoft.MvvmLight/GalaSoft.MvvmLight.AndroidSupport/Helpers/ObservableRecyclerAdapter.cs#L37) `android.support.v7.widget.RecyclerView.Adapter`, which is now [deprecated](https://developer.android.com/reference/android/support/v7/widget/RecyclerView.Adapter), we can't set the returned `Adapter` to an AndroidX `RecyclerView`.

**Comment by MichaelShapiro** [comment]
Oh, ok, I see.. so it is only for certain specific things. I actually use JimBobBennett.MvvmLight.AppCompat lib to make MVVMLight compatible with Android Support Libs (at least for navigation purposes). 

As far as I understand the Android X is still in preview, and I personally not migrating until it's all released officially. But this project, MVVMLight, seems to not being maintained anymore, unfortunately. At least I do not see any responses from Laurent for a long time now :(

**Comment by gianlucaparadise** [comment]
As far as I know, AndroidX is not in preview since 1.0.0 ([September 2018](https://developer.android.com/jetpack/androidx/releases/archive/androidx)). Support Libraries are not maintained anymore and Google [suggests](recommend using the AndroidX libraries in all new projects) to use Android X over Support Libraries.

> Android X is still in preview

Do you refer to `Xamarin.AndroidX`?

Anyway, I tried created a pull request with the migration #81.

**Comment by MichaelShapiro** [comment]
@gianlucaparadise - I am so sorry, you are correct - I meant to say that Xamarin.AndroidX is in preview. As a result I am holding off on migration in the production environment.

**Comment by gianlucaparadise** [comment]
That's true, but in my opinion, a preview version of the `MvvmLight.Platform.AndroidSupport` library migrated to AndroidX would be useful to release.

**Comment by gianlucaparadise** [comment]
I updated the pull request #81 with the latest stable AndroidX packages.

**Comment by gianlucaparadise** [comment]
Hi @MichaelShapiro ,
Is the merge of #81 planned?

Thanks.

**Comment by andrewescutia** [comment]
I need this functionality as well. Is this project dead? @gianlucaparadise did you just end up forking the project and making the changes for yourself?

**Comment by gianlucaparadise** [comment]
@andrewescutia Yes, I ended up forking the project and creating a temporary Nuget to be used until the official one is released.

In case you need them:

* My [release branch](https://github.com/gianlucaparadise/mvvmlight/tree/release/android-x)
* My [Nuget Package](https://www.nuget.org/packages/GP.MvvmLightAndroidX/)
